include/deal.II/dofs/dof_tools.h

```diff
@@ -2356,6 +2357,63 @@ namespace DoFTools
     const DoFHandler<dim, spacedim> &                   dof_handler,
     std::map<types::global_dof_index, Point<spacedim>> &support_points);
 
+  // Same as above but with support for a component mask
+
+  template <int dim, int spacedim>
+  void
+  map_dofs_to_support_points(
+    const Mapping<dim, spacedim> &                      mapping,
+    const DoFHandler<dim, spacedim> &                   dof_handler,
+    std::map<types::global_dof_index, Point<spacedim>> &support_points,
+    const ComponentMask &                               mask)
```

Comment:
why not just give the existing function the additional argument and provide a default value?